docs/user-guide/inspecting.md

```diff
@@ -11,9 +11,12 @@ Modern browsers' security settings forbid some file access which is necessary fo
 
 ## Gobview
 
+For the initial setup:
```

Comment:
I think this Markdown flavor requires an empty line before a list. At least the PR build doesn't show it correctly right now: https://goblint--993.org.readthedocs.build/en/993/user-guide/inspecting/#gobview.

```suggestion
For the initial setup:

```